client/src/modules/list/components/Items/InputBar/InputBar.jsx

```diff
@@ -35,6 +35,10 @@ class InputBar extends Component {
     this.socket = undefined;
   }
 
+  componentDidMount() {
+    document.addEventListener('keydown', this.handleEscapePress);
```

Comment:
It can't be done like this, because this handler is added always when you launch the list view and it is never removed until you exit list view. Currently, it is invoked always you press escape key wheteher or not you use this inputBar. Maybe add this handler on focus and remove on blur and componentUnMount?